servers/express.js

```diff
@@ -110,35 +42,36 @@ async function collector() {
     throw new Error('No log lines to read');
   }
   const firstTs = JSON.parse(first.value).ts;
-  
+  version = JSON.parse(first.value).entry['version'];
```

Comment:
it's important to check the version to make sure the code understands the file format. this will prevent someone from having versions of route-metrics and route-metrics-display that are out-of-sync.